libs/remote_storage/src/lib.rs

```diff
@@ -676,6 +676,7 @@ impl GenericRemoteStorage {
         to: &RemotePath,
         cancel: &CancellationToken,
     ) -> anyhow::Result<()> {
+        tracing::info!("Uploading {from_size_bytes} bytes to{to:?}");
```

Comment:
I take it this is useful enough to warrant the noise?

```suggestion
        tracing::info!("Uploading {from_size_bytes} bytes to {to:?}");
```

Reply:
This was a debug logging commit, dropped now.